.mailmap

```diff
@@ -255,13 +259,20 @@ Remzi Yang <13716567376yh@gmail.com>               remzi <13716567376yh@gmail.co
 Wang Zhuyang <hawnzug@gmail.com>                   hawnzug <hawnzug@gmail.com>
 Beta Ziliani <beta@mpi-sws.org>                    Beta Ziliani <bziliani@famaf.unc.edu.ar>
 Beta Ziliani <beta@mpi-sws.org>                    beta <beta@mpi-sws.org>
+Théo Winterhalter < >                              TheoWinterhalter
 Théo Zimmermann <theo.zimmermann@telecom-paris.fr> Théo Zimmermann <theo.zimmermann@inria.fr>
 Théo Zimmermann <theo.zimmermann@telecom-paris.fr> Theo Zimmermann <theo.zimmermann@ens.fr>
 Théo Zimmermann <theo.zimmermann@telecom-paris.fr> Théo Zimmermann <theo.zimmi@gmail.com>
 Théo Zimmermann <theo.zimmermann@telecom-paris.fr> Théo Zimmermann <theo.zimmermann@univ-paris-diderot.fr>
 Théo Zimmermann <theo.zimmermann@telecom-paris.fr> Zimmi48 <theo.zimmermann@univ-paris-diderot.fr>
 Théo Zimmermann <theo.zimmermann@telecom-paris.fr> Zimmi48 <Zimmi48@users.noreply.github.com>
 Théo Zimmermann <theo.zimmermann@telecom-paris.fr> <Théo Zimmermann <theo.zimmermann@inria.fr>
+Gabriel Scherer < >                                gasche
+Patrick Nicodemus < >                              patrick-nicodemus
+Jian Fang <hufangyu123@gmail.com>                  Rw1nd <hufangyu123@gmail.com>
+Radosław Rowicki <radrowicki@gmail.com>            radrow <radrowicki@gmail.com>
+Lucie Lahaye <github@tweqx.fr>                     Lucie <github@tweqx.fr>
+Julien Cretin <github@ia0.eu>                      ia0 <git@ia0.eu>
```

Comment:
This is using info from the github accounts: @rw1nd @radrow @lweqx @ia0
If you want some other name or pseudonym to appear in the contributor list (like the "The 59 contributors to the 8.20 version are" paragraph in https://rocq-prover.org/doc/master/refman/changes.html#version-8-20) please say so.

Reply:
Looks good to me, thanks!